File: coffee.py
import random

coffee = random.randint(0, 10)

def showSupply():
	global coffee
	print("*Coffee supply at " + str(coffee) + ".*")

def goToStarbucks():
	global coffee
	print("*Drives all the to Augusta to get frappe.*")
	print("*Returns from Augusta.*")

	coffee += 10

	showSupply()

def codeOn():
	print("*Coffee supply all good!*")
	print("*I can now finish my Python Modules.*")

	showSupply()

if (coffee <= 5):
	goToStarbucks()
else:
	codeOn()

File: coffee.php
<?php
$coffee = rand(0, 10);

function showSupply() {
	echo "*Coffee supply at ";
	print(intval($coffee));
	echo ".*\n";
}

function goToStarbucks() {
	echo "*Drives all the to Augusta to get frappe.*\n";
	echo "*Returns from Augusta.*\n";

	$coffee += 10;

	showSupply();
}

function codeOn() {
	echo "*Coffee supply all good!*\n";
	echo "*I can now finish my Minecraft PHP port.*\n";

	showSupply();
}

if ($coffee <= 5) {
	goToStarbucks();
} else {
	codeOn();
}
?>